### ISSUE TITLE
Fix path:

### ISSUE THREAD

**Issue by FridyGruder** [anthony-butcher/node-media-controller#3]
In the config variable of the index.js **windowsAppPath** is "___dirname + '\\bin\\MediaController.Program.exe'_".
If the path in question has spaces (In folder names) it will cause batch to missunderstand the command 

For example my node program is in the folder:
C:\node\my program\index.js

The **windowsAppPath** will be: _C:\node\my program\node_modules\\node-media-controller\bin\MediaController.Program.exe_

Then the command would turn out to be: _C:\node\my program\node_modules\\node-media-controller\bin\MediaController.Program.exe command next_

In batch arguments are separated by spaces and therefore this error will ensue:

Error: Command failed: C:\node\my program\node_modules\\node-media-controller\bin\MediaController.Program.exe command next
'C:\node\my' is not recognized as an internal or external command, operable program or batch file.

To overcome that issue, put some quotation marks around the path of **windowsAppPath** like these solutions:

Solution 1:
```js
var config = {
    windowsAppPath: `"${__dirname}\\bin\\MediaController.Program.exe"`,
    commands: {
        play: 'play',
        pause: 'pause',
        next: 'next',
        previous: 'previous',
        skip: 'next'
    }
};
```

Solution 2:
```js
var config = {
    `windowsAppPath: '"' + __dirname + '\\bin\\MediaController.Program.exe' + '"'`,
    commands: {
        play: 'play',
        pause: 'pause',
        next: 'next',
        previous: 'previous',
        skip: 'next'
    }
};
```

And that should resolve most of the issues concerning the path errors.

**Comment by anthony-butcher** [comment]
Was this not resolved in the PR:
https://github.com/anthony-butcher/node-media-controller/pull/2?

**Comment by FridyGruder** [comment]
> Was this not resolved in the PR: #2?

I'm not sure but if it is it's not updated on the NPM package. Maybe package.json wasn't updated..? I never uploaded packages so I don't really how it works. But I see on the second release you did fix it. You can close this
